implementation/objectoriented.tex

```diff
@@ -1,7 +1,11 @@
 \section{Object Oriented Embedded Programming}
 \label{sec:impl:oo}
 
+The interface of many of the modules defined in {\emlib} resembles that of objects found in Object Oriented programming.
+This section looks at how Object Oriented programming is used in conjunction with \gls{mmio} in embedded programming.
```

Comment:
Kanskje bytte ut "is" med "can be" i setningen "programming  ~~is~~ `can be` used in..." for å gjøre det mer klart at dette er "vår" måte tolke HWet på?